### ISSUE TITLE
www.tradingwithrayner.com - unblocked pop-up 

### ISSUE THREAD

**Issue by veronikasav** [AdguardTeam/AdguardFilters#5400]
https://www.tradingwithrayner.com/5-problems-harmonic-trading/

<details><summary>Screenshot:</summary>

![screen shot 2017-06-20 at 3 10 07 am](https://user-images.githubusercontent.com/27292111/27311061-0c4ccd98-5567-11e7-810d-a9ef0f74545d.png)

</details>

***System configuration***

Information | value
--- | ---
Operating system: | Windows / Mac OS /
Browser: | Chrome
Adguard version: | 6.2 / 1.4.1
Adguard filters: | 1 2 3 4 5 14 
Ticket ID: | 1577285

**Comment by seanl-adg** [comment]
Fixed

**Comment by snufas** [comment]
working ok

**Comment by snufas** [comment]
another website  that popups are not blocked: http://www.fairbinaryoptions.com/brokers/
i have a feeling that this software can't deal so well with popup plugins. all the filters are enabled.

**Comment by seanl-adg** [comment]
Hi, I've updated filters to block the popup in fairbinaryoptions.com. If there are still some popups unblocked, please create a new issue with a screenshot or the reproduction steps.

While we will automatically block (most of) popups caused by ads or caused by some common popup scripts, we can't automatically block popups that the site admins hard-coded in their sites. If such are reported, we will update filters with rules specific to the reported site, so please report such sites.

**Comment by snufas** [comment]
i know it is  better to creat issues, how ever i   it is eaziar to just comment  on this issue. so another 2 links: http://www.tradeciety.com/how-to-choose-the-best-indicators-for-your-trading/
is it possable that this site does not detect the addguard 
http://www.forexpeacearmy.com/forex-reviews/4884/www.topdogtrading.com?per-page=10&page=4

**Comment by Alex-302** [comment]
No, please open new issues.
